pages.ja/common/tar.md

```diff
@@ -0,0 +1,37 @@
+# tar
+
+> アーカイブ(複数のファイルやフォルダを1つのファイルに纏める)為のユーティリティー。
+> gzipやbzip2などの圧縮方法と組み合わせることが多いです。
+> 詳しくはこちら: <https://www.gnu.org/software/tar>.
+
+-  アーカイブを作成し、それをファイルに書き込む:
+
+`tar cf {{target.tar}} {{file1}} {{file2}} {{file3}}`
```

Comment:
You can translate the file names and `path/to/directory` to Japanese to bring more help non-English speakers.

Reply:
Thank you for checking.
I fixed it with the following commit.

eb6e21ab3c8e7fb2ba69eed3c1a502ae1859cbd1